test/RDA.jl

```diff
@@ -1,62 +0,0 @@
-module TestRDA
```

Comment:
What exactly is the RDA stuff? R's binary dataframe format? Should we have a stand-alone package for RDA instead of just dropping it?